From b87d3815659b86a0f8af6d5bdad73720db6ddc4d Mon Sep 17 00:00:00 2001
From: Richard Hughes <richard@hughsie.com>
Date: Tue, 20 Nov 2012 11:02:06 +0000
Subject: [PATCH] Don't dim the screen by default on battery power

It was never a good default, and we've got several enterprise distros doing
this exact change already.
---
 data/gnome-power-manager.schemas.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/gnome-power-manager.schemas.in b/data/gnome-power-manager.schemas.in
index 1ecb1e8..403c12a 100644
--- a/data/gnome-power-manager.schemas.in
+++ b/data/gnome-power-manager.schemas.in
@@ -109,7 +109,7 @@
       <applyto>/apps/gnome-power-manager/backlight/idle_dim_battery</applyto>
       <owner>gnome-power-manager</owner>
       <type>bool</type>
-      <default>true</default>
+      <default>false</default>
       <locale name="C">
         <short>Dim the screen after a period of inactivity when on battery power</short>
         <long>If the screen should be dimmed to save power when the computer is idle when on battery power.</long>
-- 
1.8.0

